plugin/src/main/kotlin/io/ktor/plugin/features/Docker.kt

```diff
@@ -58,6 +69,12 @@ abstract class DockerExtension(project: Project) {
      * Specifies an image name in form `"imageName:tag"` for an external registry.
      */
     val fullExternalImageName = externalRegistry.flatMap { it.toImage }.zipWithTag(imageTag)
+
+    /**
+     * Specifies port mappings for a `runDocker` command.
+     */
+    val portMappings: ListProperty<DockerPortMapping> = project.objects.listProperty(DockerPortMapping::class.java)
```

Comment:
could add a test?

Reply:
We don't have test infrastructure for Docker yet, I'll add a test when we have this.